src/handlers/motions/motionRewardClaimed.ts

```diff
@@ -0,0 +1,43 @@
+import { ContractEvent, MotionData } from '~types';
+import { getMotionFromDB, updateMotionInDB } from './helpers';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const {
+    contractAddress: colonyAddress,
+    args: { motionId, staker },
+  } = event;
+
+  const claimedMotion = await getMotionFromDB(colonyAddress, motionId);
+
+  if (claimedMotion) {
+    const {
+      motionData: { stakerRewards },
+    } = claimedMotion;
+
+    const updatedStakerRewards = stakerRewards.map((stakerReward) => {
+      const { address } = stakerReward;
+
+      if (address !== staker) {
+        return stakerReward;
+      }
+
+      return {
+        ...stakerReward,
+        /*
+         * This is safe because, from the front end,
+         * we claim both sides (if there were rewards on both sides)
+         * at the same time. Simply adding a flag lets us preserve the original values,
```

Comment:
In other words, we don't need an isClaimed flag for each side, because we claim both at once.